app/langswitch/LanguageSwitchController.scala

```diff
@@ -30,11 +31,13 @@ class LanguageSwitchController @Inject() (
     cc:             ControllerComponents)(
     implicit
     ec: ExecutionContext)
-  extends AbstractController(cc) {
+  extends AbstractController(cc) with Logging {
 
   import requestSupport._
 
   def switchToLanguage(language: Language): Action[AnyContent] = cc.actionBuilder { implicit request =>
+    appLogger.info("Switch language")
```

Comment:
having a log every time we hit a controller endpoint seems to be a bit excessive - we should get those logs automatically anyway. You can see the automatics logs if you search by session id in kibana

Reply:
Ah yes, ok. Some of the auto logging isn't currently running in production, but will be with deployment of main an `bootstrap-play` 7.15.0 and beyond where the default logging module is enabled by default.

Reply:
Have removed logging that only capture the controller endpoint hits (where no journey information is imparted).

Would you like me to also remove all the logging which capture the controller endpoint hits with journeyId info? These are currently only at INFO level so won't make it to production kibana anyway.

I find them fairly helpful to track the logging in the terminal when I run the app locally, but they are a bit of a burden to maintain the convention in the code.

Reply:
I have decided to leave in logging of controller end point calls with journeyId info at INFO level as I find them useful when running the service locally. They will not appear in production as we are setting the level there to WARN.